src/core/config/config.go

```diff
@@ -217,7 +217,11 @@ func SelfRegistration() (bool, error) {
 
 // RegistryURL ...
 func RegistryURL() (string, error) {
-	return cfgMgr.Get(common.RegistryURL).GetString(), nil
+	url := os.Getenv("REGISTRY_URL")
```

Comment:
what's the purpose to modify it?

Reply:
The registry URL cannot be read before the database started if use the config manager. This isn't needed